susemanager-utils/testing/docker/master/uyuni-master-pgsql-4eclipse/Dockerfile

```diff
@@ -8,7 +8,8 @@ MAINTAINER Michael Calmer "Michael.Calmer@suse.com"
 #RUN zypper --non-interactive in -n spacewalk-admin susemanager-schema smdba
 RUN zypper --non-interactive in -n spacewalk-admin smdba
 
-RUN zypper --non-interactive in /root/susemanager-schema-4.1.8-1.2.uyuni.noarch.rpm
+#RUN zypper --non-interactive in /root/susemanager-schema-4.1.8-1.2.uyuni.noarch.rpm
```

Comment:
See my previous comment. Maybe we should just remove this, and avoid adding the RPM to the image.

Reply:
This is only for the "4elclipse" containers. This beast should come up as fast as possible.

Reply:
I would remove instead of commenting, unless you have a good reason to leave it commented.

Reply:
done